.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Question
+about: Ask a question about how to use T3
+title: ''
+labels: 'Type: Question'
+assignees: ''
+
+---
+
```

Comment:
Can we add a phrase such as, “<-- Thanks for your interest in T3! Please first check the online documentation at <INSERT_URL_HERE> to see if it answers your question. Also, try searching GitHub issues to see if someone has asked your question before. --> 

For some reason, I get a 404 error when trying to load T3’s documentation at the moment so I can’t access the url. Does the page load for you?

Could we add similar short headers that encourage the user to search GitHub for the bug report and feature suggestion?

Reply:
Good idea! Added